test/cases/file-selector-button.css

```diff
@@ -0,0 +1,3 @@
+::file-selector-button {
```

Comment:
You need to call `file-selector-button` test case in `test/autoprefixer.test.ts` (see hacks block there)

Reply:
Think I've added it to the correct location